patch/attiny84.yaml

```diff
@@ -0,0 +1,221 @@
+_include:
+  - "common/ac.yaml"
+  - "common/port.yaml"
+  - "common/wdt.yaml"
+
+  - "timer/attiny84.yaml"
+
+AC:
+  _modify:
+    DIDR0:
+      description: "Digital Input Disable Register 0"
+  DIDR0:
+    _delete:
+      - ADC0D
+    _modify:
+      ADC1D:
+        description: "ADC1 (AIN0) Digital input buffer disable"
+    _add:
+      ADC2D:
+        description: "ADC2 (AIN1) Digital input buffer disable"
+        bitOffset: 2
+        bitWidth: 1
+        access: read-write
+ADC:
+  ADCSRA:
+    ADPS:
+      _replace_enum:
+        PRESCALER_2: [1, "Prescaler Value 2"]
+        PRESCALER_4: [2, "Prescaler Value 4"]
+        PRESCALER_8: [3, "Prescaler Value 8"]
+        PRESCALER_16: [4, "Prescaler Value 16"]
+        PRESCALER_32: [5, "Prescaler Value 32"]
+        PRESCALER_64: [6, "Prescaler Value 64"]
+        PRESCALER_128: [7, "Prescaler Value 128"]
+  ADCSRB:
+    ADTS:
+      _replace_enum:
+        FREE: [0, "Free Running mode"]
+        AC: [1, "Analog Comparator"]
+        INT0: [2, "External Interrupt Request 0"]
+        TC0_CMA: [3, "Timer/Counter0 Compare Match A"]
+        TC0_OVF: [4, "Timer/Counter0 Overflow"]
+        TC0_CMB: [5, "Timer/Counter0 Compare Match B"]
+        PCIR: [6, "Pin Change Interrupt Request"]
+  ADMUX:
+    _add:
+      MUX:
+        description: "Analog Channel and Gain Selection Bits"
+        bitOffset: 0
+        bitWidth: 6
+        access: read-write
+      REFS:
+        description: "Reference Selection Bits"
+        bitOffset: 6
+        bitWidth: 2
+        access: read-write
+    MUX:
+      _write_constraint: enum
```

Comment:
Does this work?  I think `_write_constraint` can only be used in an `_modify` block?

Reply:
I don't know, I'll go double check.

Reply:
You're right, it's not relevant here.  I can skip the `_replace_enum` as well, since the field is new.